docs/manuals/core/entities/pipeline-mgt.md

```diff
@@ -65,20 +65,39 @@ pipeline == pipeline_retrieved
 
 Here the two variables `pipeline` and `pipeline_retrieved` are equal.
 
-# Get pipeline by config id
+# Get pipelines by config id
 
 A pipeline can also be retrieved from a scenario by accessing the pipeline's config_id of the scenario.
```

Comment:
"config_id" -> "configuration identifier"

Reply:
as said above, I still think config_id is better as it's clear and straight to the point :D but if you think otherwise, I'm open to changing it to configuration identifier :D